src/platform/tls_openssl.c

```diff
@@ -841,6 +851,84 @@ CxPlatTlsSecConfigCreate(
                 Status = QUIC_STATUS_TLS_ERROR;
                 goto Exit;
             }
+        } else if (CredConfig->Type == QUIC_CREDENTIAL_TYPE_CERTIFICATE_PKCS12) {
+            BIO* bio = BIO_new(BIO_s_mem());
```

Comment:
Can `BIO_s_mem()` fail? What happens if it does? Also, we use capitals for the first letters of our variables.

Reply:
no.
```c
const BIO_METHOD *BIO_s_mem(void)
{
    return &mem_method;
}
```
`mem_method` is `static const`.